package.json

```diff
@@ -8,11 +8,13 @@
         "dist"
     ],
     "scripts": {
-        "prepare": "npm run build",
```

Comment:
Why change this to `prepack`?

Reply:
Otherwise this also runs on npm install, effectively running the entire build twice during CI. Why would a developer to Sherlock want to run a full build on a npm install?

Reply:
prepack runs on pack and publish and seems the more obvious choice

Reply:
IC, npm command suggested changing it to "prepare". Not really looked into it before.